### ISSUE TITLE
[Content New] Accessibility Tutorial for Astronomy Projects

### ISSUE THREAD

**Issue by kakirastern** [astropy/astropy-tutorials#456]
As initiated by Tim Rhue on the Astropy slack `general` channel, and recorded in a thread [here](https://astropy.slack.com/archives/C067V74GK/p1597084825363000), Learn Astropy will volunteer to organize our efforts around drafting and thereby completing an accessibility tutorial for astronomy projects using `Python` as the programming language. The focus will be mostly on the visual aspect of accessibility such as ones arising from but not limited to color blindness and color weakness (i.e., forms of color vision deficiency) that impacts one's ability to see color contrast compared to the average population. Ideally in-depth discussions surrounding best practices regarding color scheme use or more specifically colormap usage should be included for a practical guide to be adopted by the larger astronomy community. Some web accessibility tools for gauging color contrast can also be introduced as a tool to aid such practices.

**Comment by kakirastern** [comment]
https://webaim.org/intro/

**Comment by kakirastern** [comment]
https://webaim.org/resources/contrastchecker/

**Comment by kakirastern** [comment]
https://matplotlib.org/3.2.1/tutorials/colors/colormaps.html#color-vision-deficiencies

**Comment by kakirastern** [comment]
https://www.color-blindness.com/

**Comment by kakirastern** [comment]
https://developers.google.com/web/updates/2020/11/cvd
